test/gerkin/features/steps/db_datasets.js

```diff
@@ -0,0 +1,65 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { header, localhost } = require('./helpers/helpers');
+
+let specDataRead;
+
+const baseUrl = `${localhost}databases`;
+
+Before(() => {
+  specDataRead = pactum.spec();
+});
+
+// Background: The user wants to display information about all Digital Registries databases
+Given(
+  'The user wants to display information about all Digital Registries databases', () => {
+  return 'The user wants to display information about all Digital Registries databases';
+  }
+);
+
+// Scenario: The user receives information about Digital Registries databases
+When(
+  'The user triggers an action with a valid payload to display information about Digital Registries databases', () => {
+    specDataRead
+      .get(`${baseUrl}`)
+      .withHeaders(`${header.key}`, `${header.value}`);
+  }
+);
+
+Then('The user received a list of Digital Registries databases', async () => {
+  await specDataRead.toss();
+  specDataRead.response().should.have.status(200);
```

Comment:
In the titile is "The user received a list of..." - is it possible to check if user really receive those list? What is API response?